src/payment/klarna.md

```diff
@@ -8,20 +8,20 @@ Klarna lets your customers choose how they want to pay and supports the purchase
 
 ### New features
 
-- Added support for Australia and New Zealand markets
-- Improved configuration: Changed scope of API Configuration from `Store View` to `Website`; updated section name and field label changes.
+- Added support for On-Site Messaging
 
 ## Klarna payment methods
 
-- [**Klarna. Installments**][3] - Gives your customers an interest-free alternative to credit with four installments.
+- [**Klarna. 4 Installments**][3] - Gives your customers an interest-free alternative to credit with four installments.
 - [**Klarna. Pay in 30 days**][4] - Gives your customers an extra 30 days to pay.
 - [**Klarna. Financing**][5] - Gives your customers the option to pay over 3 to 36 months.
 
 ## Klarna solutions
 
-- [**Klarna. Checkout**][6] - A complete online checkout solution and smooth shopping experience that increases sales and customer loyalty.
+- [**Klarna. Checkout**][6] - A complete online checkout solution and smoooth shopping experience that increases sales and customer loyalty.
```

Comment:
I'm enclosing this in quotes because although it is part of your branding, it looks like a typo within the context of the user guide,
```suggestion
- [**Klarna. Checkout**][6] - A complete online checkout solution and "smoooth" shopping experience that increases sales and customer loyalty.
```